commit ef45cc5ab4dc50a88506e0360cf360af29235aa4
Author: Christine Caulfield <ccaulfie@redhat.com>
Date:   Tue Dec 3 09:56:25 2013 +0000

    config: Read logging attributes correctly
    
    an @ sign was missing when reading the path
    /cluster/logging/logging_daemon[@name=\"%s\"]/%s which caused
    logging attributes set ub cluster.conf to not be applied. This
    patches adds the @ and fixes the problem.
    
    rhbz#980575
    
    Signed-off-by: Christine Caulfield <ccaulfie@redhat.com>

diff --git a/config/libs/libccsconfdb/extras.c b/config/libs/libccsconfdb/extras.c
index b0cf55b..7d187c9 100644
--- a/config/libs/libccsconfdb/extras.c
+++ b/config/libs/libccsconfdb/extras.c
@@ -284,7 +284,7 @@ static void create_daemon_path(const char *name, const char *field, char *path)
 {
 	memset(path, 0, PATH_MAX);
 	snprintf(path, PATH_MAX,
-		 "/cluster/logging/logging_daemon[@name=\"%s\"]/%s",
+		 "/cluster/logging/logging_daemon[@name=\"%s\"]/@%s",
 		 name, field);
 }
 
